tests/catch/unit/module/hipModuleLoad.cc

```diff
@@ -50,7 +50,10 @@ TEST_CASE("Unit_hipModuleLoad_Negative_Parameters") {
     HIP_CHECK_ERROR(hipModuleLoad(&module, "non existent file"), hipErrorFileNotFound);
   }
 
+// Disabled for AMD due to defect
+#if HT_NVIDIA
```

Comment:
ticket?

Reply:
Yes. I've updated the PR with the ticket IDs added in the comments.